examples/chart/teleport/Chart.yaml

```diff
@@ -1,6 +1,8 @@
 name: teleport
-version: 0.0.4
-description: Teleport Enterprise
+apiVersion: v1
+version: 0.0.5
+description: Setup a Teleport Cluster
```

Comment:
I think this should probably be a more verbose, proper description like "Teleport provides a secure remote access solution for developers that doesn't get in the way."

The reason is because this string will appear in the output of `helm search repo teleport`. Here's an example for what the descriptions are for Drone-focused charts:

```bash
$ helm search repo drone
NAME                          	CHART VERSION	APP VERSION	DESCRIPTION                                       
drone/drone                   	0.1.6        	1.8.1      	Drone is a self-service Continuous Delivery pla...
drone/drone-kubernetes-secrets	0.1.0        	1.0.0      	A Kubernetes Secrets extension for Drone          
drone/drone-runner-kube       	0.1.3        	1.0.0      	The Drone Kubernetes runner launches builds in ...
stable/drone                  	2.7.2        	1.6.5      	Drone is a Continuous Delivery system built on ..
```